src/components/Footer.jsx

```diff
@@ -0,0 +1,16 @@
+function Footer() {
+    return (
+        <footer style={{
+            backgroundColor: '#f1f1f1',
+            textAlign: 'center',
+            position: 'fixed',
+            left: '0',
+            bottom: '0',
+            width: '100%',
+        }}>
+            <p style={{ fontSize: '0.8em' }}>© 2024 My ToDo App. All rights reserved.</p>
```

Comment:
It is a better approach instead of hardcoded `2024` use a Date class to create current year

Reply:
fixed